### DISCUSSION TITLE
go module

### DISCUSSION THREAD

**Discussion by PatriotBo** [traefik/yaegi#1506] [Q&A] [answered]
Will yaegi support go module?  It is so troublesome when using 3rd party lib in scripts

**Comment by mvertes** [comment, marked as answer]
Yes, this is long overdue. It has been slowed by the fact that most of the module machinery is outside of the Go standard library, and we do not want to add any external dependencies. I think that it should still be possible to come up with an internal very simple resolver just to get the source package location under `GOMODCACHE` (i.e `~/go/pkg/mod/...`), and skip everything else.
